src/enumerable.cr

```diff
@@ -558,6 +558,25 @@ module Enumerable(T)
     raise Enumerable::NotFoundError.new
   end
 
+  # Yields each value until the first truthy block result and returns that result.
+  #
+  # Accepts an optional parameter `if_none`, to set what gets returned if
+  # no element is found (defaults to `nil`).
+  #
+  # ```
+  # [1, 2, 3, 4].find { |i| i > 2 }     # => 3
+  # [1, 2, 3, 4].find { |i| i > 8 }     # => nil
+  # [1, 2, 3, 4].find(-1) { |i| i > 8 } # => -1
```

Comment:
```suggestion
  # [1, 2, 3, 4].find_value { |i| i > 2 }     # => true
  # [1, 2, 3, 4].find_value { |i| i > 8 }     # => nil
  # [1, 2, 3, 4].find_value(-1) { |i| i > 8 } # => -1
```

Reply:
Ha! Thanks! I was happy I remembered to at least copy the docs over … and then immediately forgot about updating them. 😄